extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -169,6 +180,70 @@ export class WizardController {
 			}));
 	}
 
+	private async createLoginWizard(stateModel: MigrationStateModel): Promise<void> {
+		const serverName = (await stateModel.getSourceConnectionProfile()).serverName;
+		this._wizardObject = azdata.window.createWizard(
+			loc.LOGIN_WIZARD_TITLE(serverName),
+			'LoginMigrationWizard',
+			'wide');
+
+		this._wizardObject.generateScriptButton.enabled = false;
+		this._wizardObject.generateScriptButton.hidden = true;
+		const targetSelectionPage = new LoginMigrationTargetSelectionPage(this._wizardObject, stateModel);
+		const loginSelectorPage = new LoginSelectorPage(this._wizardObject, stateModel);
+		const migrationStatusPage = new LoginMigrationStatusPage(this._wizardObject, stateModel);
+
+		const pages: MigrationWizardPage[] = [
+			targetSelectionPage,
+			loginSelectorPage,
+			migrationStatusPage
+		];
+
+		this._wizardObject.pages = pages.map(p => p.getwizardPage());
+
+		const wizardSetupPromises: Thenable<void>[] = [];
+		wizardSetupPromises.push(...pages.map(p => p.registerWizardContent()));
+		wizardSetupPromises.push(this._wizardObject.open());
+
+		this._model.extensionContext.subscriptions.push(
+			this._wizardObject.onPageChanged(
+				async (pageChangeInfo: azdata.window.WizardPageChangeInfo) => {
+					const newPage = pageChangeInfo.newPage;
+					const lastPage = pageChangeInfo.lastPage;
+					this.sendPageButtonClickEvent(pageChangeInfo)
+						.catch(e => logError(
+							TelemetryViews.LoginMigrationWizardController,
+							'ErrorSendingPageButtonClick', e));
+					await pages[lastPage]?.onPageLeave(pageChangeInfo);
+					await pages[newPage]?.onPageEnter(pageChangeInfo);
+				}));
+
+		this._wizardObject.registerNavigationValidator(async validator => {
+			// const lastPage = validator.lastPage;
```

Comment:
is the commented code still needed or can it be cleaned up

Reply:
this was copied from the launch data migration wizard. The intent behind it was preventing navigation back and forth when it is not allowed. I ended up using a different approach to manage this and so I can remove this. 

It looks like that portion in launch data migration wizard isn't being used either, so I'll remove that as well